hardware_interface/src/component_parser.cpp

```diff
@@ -593,9 +594,4 @@ std::vector<HardwareInfo> parse_control_resources_from_urdf(const std::string &
   return hardware_info;
 }
 
-bool parse_bool(const std::string & bool_string)
-{
-  return bool_string == "true" || bool_string == "True";
-}
-
```

Comment:
@christophfroehlich Can you please also remove it's definition from the header file as well

https://github.com/ros-controls/ros2_control/blob/e149646d3f2595f269cfa4e1cd0681abde89ee69/hardware_interface/include/hardware_interface/component_parser.hpp#L36-L37

Reply:
sure, good catch.